### ISSUE TITLE
Question about the paper

### ISSUE THREAD

**Issue by MohammadHossein-Bahari** [OATML/RHO-Loss#2]
Hello,
Nice paper!
I have two questions:
1- I couldn't understand how the method prioritizes ''good samples' **In Eq 3**'? Actually I don't see how different an outlier is with a good sample and I guess both will lead to high training and IL losses. Since a good sample is one that hasn't been learned yet, it has high training and IL loss (it should have high IL loss as it hasn't been learned on the training data, so no reason that it has been learned on the hold-out data). 
2- Can you explain ''iii) Less relevant points'' in section 3 on Eq 3? Since both the training set and holdout set are from p_true, they should both lead to the same loss values (i.e., either low or high) for outlier samples.

I appreciate your answer in advance!

**Comment by mtrazzak** [comment]
Sorry about the delay! Can you drop us an email and we'd be happy to answer them there!
